Managers/AudioMan.cpp

```diff
@@ -1084,7 +1088,8 @@ bool AudioMan::SetSoundAttenuation(Sound *pSound, float distance)
 		Mix_SetDistance(pSound->m_LastChannel, (255 * distance));
 #elif __USE_SOUND_GORILLA
 		if (pSound->m_LastChannel >= 0)
-			ga_handle_setParamf(m_SoundChannels[pSound->m_LastChannel], GA_HANDLE_PARAM_GAIN, MAX_VOLUME * (1.0f - distance));
+			ga_handle_setParamf(m_SoundChannels[pSound->m_LastChannel], GA_HANDLE_PARAM_GAIN, m_SoundsVolume * (1.0f - distance));
```

Comment:
As above, this should probably include MAX_VOLUME in its formula, i.e.
ga_handle_setParamf(m_SoundChannels[pSound->m_LastChannel], GA_HANDLE_PARAM_GAIN, MAX_VOLUME * m_SoundsVolume * (1.0f - distance));